test/spark/test_spark_shaping.py

```diff
@@ -217,3 +217,22 @@ def test_swapaxes(sc):
     assert allclose(b.swapaxes(0, 1).toarray(), b.toarray().swapaxes(0, 1))
     assert allclose(b.swapaxes(2, 3).toarray(), b.toarray().swapaxes(2, 3))
 
+def test_reshape(sc):
+
+    old_shape = (6, 10, 4, 12)
+    a = arange(prod(old_shape)).reshape(old_shape)
+    b = array(a, sc, axis=[0, 1])
```

Comment:
swap `(0,1)` for `[0,1]`, I know it's the same, but we've used the tuple version throughout the tests